deploy/env/e2e-tests.yaml

```diff
@@ -1,6 +1,13 @@
 registration-service:
   environment: 'e2e-tests'
   replicas: 2
+  verification:
+    enabled: 'true'
+    excluded-email-domains: 'redhat.com,acme.com'
+  twilio:
+    account-sid: ''
```

Comment:
do we need to add these here if the default is empty?

Reply:
I didn't intend to add these changes yet.. I've commented them out until the e2e tests are running successfully.